webpack.config.js

```diff
@@ -17,20 +19,50 @@ module.exports = (env) => {
           use: ["babel-loader"],
         },
         {
-          test: /\.(css|sass)$/i,
-          use: ["style-loader", "css-loader", "sass-loader"],
+          test: /\.css$/i,
+          use: [MiniCssExtractPlugin.loader, "css-loader"],
```

Comment:
You should've enabled mini-css-extract-plugin just for production build. For development, `style-loader` will just be faster.

Reply:
fixed